enos/modules/vault_cluster/main.tf

```diff
@@ -81,22 +105,13 @@ resource "enos_bundle_install" "vault" {
   }
 }
 
-module "install_packages" {
-  source = "../install_packages"
-  depends_on = [
-    enos_bundle_install.vault, // Don't race for the package manager locks with vault install
-  ]
-
-  hosts    = var.target_hosts
-  packages = var.packages
-}
-
 resource "enos_consul_start" "consul" {
   for_each = enos_bundle_install.consul
 
   bin_path = local.consul_bin_path
   data_dir = var.consul_data_dir
   config = {
+    bind_addr        = "{{ GetPrivateInterfaces | include \"type\" \"IP\" | sort \"default\" |  limit 1 | attr \"address\"}}"
```

Comment:
We put a comment here about why we're doing this at all. IIRC it's because Amazon Linux has a default docker listener that messes with consul by default. This is a `go-sockaddr` template that helps consul get the correct addr in all of our default cases.